modules/tinymce/src/themes/silver/test/ts/browser/window/SilverInlineDialogTest.ts

```diff
@@ -94,11 +114,14 @@ describe('browser.tinymce.themes.silver.window.SilverInlineDialogTest', () => {
   const openDialog = (
     editor: Editor,
     params: WindowParams,
-    dialog: Dialog.DialogSpec<{ fred: string }> & { size?: Dialog.DialogSize } = dialogSpec
+    dialog: Dialog.DialogSpec<{ fred: string }> & { size?: Dialog.DialogSize } = dialogSpec,
+    iframe: boolean = false
   ) => {
     const api = DialogUtils.openWithStore(editor, dialog, params, store);
-    const { fred } = api.getData();
-    assert.deepEqual({ fred }, { fred: 'said hello pebbles' }, 'Initial data');
+    if (!iframe) {
```

Comment:
I think we should have a separate function called `openDialogWithIframe` or even just call `const api = DialogUtils.openWithStore(editor, dialog, params, store);` in the block below. Adding conditionals and 'defaulted' parameters will make this test file harder to maintain